tachyon/c/zk/air/sp1/baby_bear_poseidon2_two_adic_fri.h

```diff
@@ -79,16 +70,17 @@ tachyon_sp1_baby_bear_poseidon2_two_adic_fri_coset_lde_batch(
     tachyon_baby_bear* extended_values, tachyon_baby_bear shift);
 
 /**
- * @brief Commits to the mixed matrix created by
- * tachyon_sp1_baby_bear_poseidon2_two_adic_fri_coset_lde_batch.
+ * @brief Commits to the lde vector.
  *
  * @param pcs A pointer to the two adic fri.
+ * @param lde_vec A pointer to lde vector.
```

Comment:
```suggestion
 * @param lde_vec A pointer to the lde vector.
```
How about like this for consistency?